content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu agent instances](#monitor-your-sensu-agent-instances)
+- [Monitor your Sensu API instances](#monitor-your-sensu-api-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu. 
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard instances.
+
+To completely monitor a Sensu stack [Sensu backend, agent, external etcd cluster (if used)], you will need to have at least one other independent Sensu instance.
```

Comment:
We may want to take out the use of the word stack here, as that was really about the high number of components needed to run SE Classic (SE, SE dashboard, rabbitmq, redis). Where now the requirement for most users is going to be just the Sensu Backend with the internal etcd.

We may want to eventually set up a section for the other possible setups (external, postgres).

Reply:
Thank you -- I will update this and add placeholders for External etcd and PostgreSQL.